docs/helm-broker/docs/03-04-bundles-docs.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Bundles docs
+type: Details
+---
+
+## Documentation for ServiceClasses provided by bundle
+
+The Helm Broker in the provisioning process pushes the `addonRepositoryURL` variable into installed chart. With this feature a bundle which provides its own ServiceClasses e.g. `Azure Broker` can also install a documentation for them.
+
+If the bundle provides ServiceClasses and you want to have a documentation for it, you should create a `docs.yaml` inside the bundle's chart with the following structure:
+```
+apiVersion: cms.kyma-project.io/v1alpha1
+kind: ClusterDocsTopic
+metadata:
+  labels:
+    cms.kyma-project.io/view-context: service-catalog
+  name: {ServiceClass ID}
+spec:
+  displayName: "Slack Connector Add-on"
```

Comment:
Should we use placeholder rather than examples?

Reply:
@klaudiagrz What do you think?

Reply:
@klaudiagrz ? :)